specification/apicenter/data-plane/ApiCenter.DataApi/preview/2024-02-01-preview/azure.apicenter.dataapi.json

```diff
@@ -0,0 +1,1644 @@
+          }
+        ]
+      }
+    },
+    "Models.EnvironmentOnboardingModel": {
+      "type": "object",
+      "description": "Environment Onboarding Model",
+      "properties": {
+        "instructions": {
+          "type": "string",
+          "description": "Instructions how to onboard to the environment.",
+          "maxLength": 1000
+        },
+        "developerPortalUris": {
+          "type": "array",
+          "description": "Developer portal URIs of the environment.",
+          "items": {
+            "type": "string",
+            "format": "uri"
+          }
+        }
+      },
+      "required": [
+        "instructions",
+        "developerPortalUris"
+      ]
+    },
+    "Models.EnvironmentServer": {
+      "type": "object",
+      "description": "Environment Server",
+      "properties": {
+        "type": {
+          "type": "string",
+          "description": "Type of the server that represents the environment."
+        },
+        "managementPortalUris": {
+          "type": "array",
+          "description": "URIs of the server's management portal.",
+          "items": {
+            "type": "string",
+            "format": "uri"
+          }
+        }
+      },
+      "required": [
+        "type",
+        "managementPortalUris"
+      ]
+    },
+    "Models.ExportSpecification": {
+      "type": "object",
+      "description": "Export specification",
+      "properties": {
+        "value": {
+          "type": "string",
+          "description": "The result of the specification export."
+        },
+        "format": {
+          "$ref": "#/definitions/Models.SpecFormat",
+          "description": "The result format of the specification export."
+        }
+      },
+      "required": [
+        "value",
+        "format"
+      ]
+    },
+    "Models.SpecFormat": {
+      "type": "string",
+      "description": "Export specification format",
+      "enum": [
+        "inline",
+        "link"
+      ],
+      "x-ms-enum": {
+        "name": "SpecFormat",
+        "modelAsString": true,
+        "values": [
+          {
+            "name": "Inline",
+            "value": "inline",
+            "description": "Inline format"
+          },
+          {
+            "name": "Link",
+            "value": "link",
+            "description": "Link format"
+          }
+        ]
+      }
+    },
+    "PagedApi": {
+      "type": "object",
+      "description": "Paged collection of Api items",
+      "properties": {
+        "value": {
+          "type": "array",
+          "description": "The Api items on this page",
+          "items": {
+            "$ref": "#/definitions/Models.Api"
+          },
+          "x-ms-identifiers": []
+        },
+        "nextLink": {
+          "type": "string",
+          "format": "uri",
+          "description": "The link to the next page of items"
+        }
+      },
+      "required": [
+        "value"
+      ]
+    },
+    "PagedApiDefinition": {
```

Comment:
Should we call it `[Entity]Collection` instead of `Paged[Entity]`? A collection of items is not necessarily paged.

Reply:
There are two standard operations provided by the Azure Core standard operations library.
We are currently using ResourceList, which always returns a paginated collection:
https://azure.github.io/typespec-azure/docs/libraries/azure-core/reference/interfaces#Azure.Core.ResourceOperations.ResourceList

There is also NonPagedResourceList, which always returns a non-paginated collection:
https://azure.github.io/typespec-azure/docs/libraries/azure-core/reference/interfaces#Azure.Core.Foundations.NonPagedResourceList

Should we use the NonPagedResourceList operation? I am not sure if there is a type that can return both kind of results.

Reply:
@azaslonov Following up on this comment before resolving.